topic/timeseries/timeseries-queries-and-visualization.ipynb

```diff
@@ -610,11 +661,10 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "from datetime import datetime\n",
+    "import datetime as dt\n",
```

Comment:
This, and the corresponding adjustments, are probably the only relevant changes to code cells, all the other changes are only about prose.